### PR TITLE
[MIG] account_payment_mode: Migration to 13.0

### PR DESCRIPTION
**Pull request by lembregtse**
First module to support sepa migration.

Basic changes only.

### CODE REVIEW

File: account_payment_mode/models/account_payment_mode.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    important changes. It also replaces the object payment.method
+    of the module sale_payment_method of OCA/e-commerce"""
+    _name = "account.payment.mode"
+    _description = 'Payment Modes'
+    _order = 'name'
+
+    name = fields.Char(required=True, translate=True)
+    company_id = fields.Many2one(
+        'res.company', string='Company', required=True, ondelete='restrict',
+        default=lambda self: self.env['res.company']._company_default_get(
```
Comment (astirpe) [inline account_payment_mode/models/account_payment_mode.py]: `company_default_get()` is deprecated in V13.

Use:

```python
default=lambda self: self.env.company
```

**Review by astirpe — CHANGES_REQUESTED** [review]
Please rebase and run `pre-commit run -a`, as per the [Migration guide](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0)

**Review by angelmoya — COMMENTED** [review]
Code LGTM, runbot is not working

**Comment by pedrobaeza** [comment]
Are you going to fix this?

**Comment by rven** [comment]
> Are you going to fix this?

I have done the rebase, [IMP] account_payment_mode: black, isort, and the remark of @astirpe

File: account_payment_mode/models/account_payment_mode.py (hunk 1 above)
Comment (rven) [reply account_payment_mode/models/account_payment_mode.py]: Done!

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Let's merge it as base dependency for lot of modules

/ocabot merge

File: account_payment_mode/views/account_payment_mode.xml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                <field name="fixed_journal_id"
+                    attrs="{'invisible': [('bank_account_link', '!=', 'fixed')], 'required': [('bank_account_link', '=', 'fixed')]}"
+                    domain="[('company_id', '=', company_id), ('type', '=', 'bank')]"
+                />
+                <field name="variable_journal_ids"
+                    attrs="{'invisible': [('bank_account_link', '!=', 'variable')], 'required': [('bank_account_link', '=', 'variable')]}"
+                    domain="[('company_id', '=', company_id), ('type', '=', 'bank')]"
+                    widget="many2many_tags"/>
+            </group>
+            <group string="Note" name="note">
```
Comment (CasVissers-360ERP) [inline account_payment_mode/views/account_payment_mode.xml:25]: image
Translation seems not correctly visible for notes field

**Comment by rven** [comment]
@CasVissers The translation is defined in the po files
image
It seems to be an Odoo problem, because when opening a supplier (partner) the tab pages, prompts are also not translated.
image

**Comment by CasVissers-360ERP** [comment]
@rven 
Wat I actually mean is the alignment of the view seems incorrect, only part of EN is visible.

Anyway it's a bit nitpicky and since already merged no need for an issue.

**Pull request merged**